packages/core/src/Stores/client-store.js

```diff
@@ -2416,6 +2438,12 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    responseCTraderAvailableAccounts(response) {
+        if (!response.error) {
+            this.ctrader_available_accounts = response.trading_platform_accounts;
+        }
```

Comment:
This needs error handling in case an error occurred. 
Maybe returning empty array!